roboflow/core/version.py

```diff
@@ -344,11 +365,14 @@ def deploy(self, model_type: str, model_path: str) -> None:
 
         model = torch.load(os.path.join(model_path, "weights/best.pt"))
 
-        class_names = []
-        for i, val in enumerate(model["model"].names):
-            class_names.append((val, model["model"].names[val]))
-        class_names.sort(key=lambda x: x[0])
-        class_names = [x[1] for x in class_names]
+        if isinstance(model["model"].names, list):
+            class_names = model["model"].names
+        else:
+            class_names = []
+            for i, val in enumerate(model["model"].names):
```

Comment:
`enumerate` works on an iterable, what should be the type expected here? Since we checked for `list` before

Reply:
In this case I think it is a dict